AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -20,20 +20,26 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
     private var currentAudioState = SessionStateControllerAction.initialize
     private var currentAudioStatus = MeetingSessionStatusCode.ok
     private let realtimeObservers = ConcurrentMutableSet()
+    private let logger: Logger
 
     private let audioLock: NSLock
 
     init(audioClient: AudioClient, clientMetricsCollector: ClientMetricsCollector,
-         audioClientLock: NSLock, configuration: MeetingSessionConfiguration) {
+         audioClientLock: NSLock, configuration: MeetingSessionConfiguration, logger: Logger) {
         self.audioClient = audioClient
         self.clientMetricsCollector = clientMetricsCollector
         self.configuration = configuration
+        self.logger = logger
         audioLock = audioClientLock
         super.init()
         audioClient.delegate = self
     }
 
     public func audioClientStateChanged(_ audioClientState: audio_client_state_t, status: audio_client_status_t) {
+        logger.debug(debugFunction: {
+            return "AudioClient State: \(audioClientState.rawValue) Status: \(status.rawValue)"
+        })
```

Comment:
Is the debug logging for us to troubleshoot SDK issues or for developers to troubleshoot theirs? If latter, how are they going to comprehend the raw value?

Reply:
This is to help us troubleshoot issues. Builders don't have to know what these values mean since our Media library is close source. When issues happens, they can send debug builds to customers can send logs back to us.